src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -49,6 +59,14 @@ ccjRequestedForDefendantClaimDetails %}
           {% else %}
             {{ claimRejectedFreeMediationYESForClaimantClaimDetails(claim) }}
           {% endif %}
+          {% elif claim.response.responseType === domain.ResponseType.PART_ADMISSION %}
```

Comment:
Either wrong or misaligned as 'elif' should not be the same indentation as 'endif'

Reply:
Please correct alignment.